src/security-rules/security-rules-api-client.ts

```diff
@@ -49,11 +114,29 @@ export class SecurityRulesApiClient {
    * @param {string} name Full qualified name of the resource to get.
    * @returns {Promise<T>} A promise that fulfills with the resource.
    */
-  public getResource<T>(name: string): Promise<T> {
+  private getResource<T>(name: string): Promise<T> {
     const request: HttpRequestConfig = {
       method: 'GET',
       url: `${this.url}/${name}`,
     };
+    return this.sendRequest<T>(request);
+  }
+
+  private getRulesetName(name: string): string {
```

Comment:
any reason we do this for rulesets but not for releases?

Reply:
With Rulesets we will need this name processing functionality for deleteRuleset (implemented in a future PR) as well. I don't think there's any other API that requires release name processing. If it comes up, I'll refactor accordingly.

Reply:
This looks right to me; we don't need to look up a release by name.